cmd/pperf/main.go

```diff
@@ -17,15 +18,15 @@ func main() {
 
 	flag.Parse()
 
-	results := pperf.Pperf(pperf.API{
+	results, err := pperf.Run(pperf.API{
 		Server:    *server,
 		Target:    *target,
 		Port:      *port,
 		Seconds:   *seconds,
 		Interface: *ifname,
 	})
-	if results.Err != nil {
-		fmt.Println(results.Err.Error())
+	if err != nil {
```

Comment:
Recommend formatting this as a json output to keep consistency with results output.  Alternatively, a non-zero exit code would be sufficient to allow the error output to be determined.